### PR TITLE
new CMS guide: Craft CMS

### PR DESCRIPTION
**Pull request by olets**
#### Description (required)

Adds a Craft CMS page in the CMSes' section.

#### Related issues & labels (optional)

- Requested in https://mastodon.social/@sarah11918/112586867718861380

- Suggested label:
  - add new content
  - 3rd-party guide

https://astro.build/chat: `@olets`

### CODE REVIEW

File: src/content/docs/en/guides/cms/craft-cms.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+i18nReady: true
+stub: true
+---
+
+[Craft CMS](https://craftcms.com/) is a flexible open source CMS with an accessible authoring experience. It includes its own front end, but can also be used as a headless CMS to provide content to your Astro project.
+
+## Official Resources
+
+- Check out the official Craft CMS [GraphQL API guide](https://craftcms.com/docs/5.x/development/graphql.html)
+- The official documentation for Craft's [`headlessMode` config setting](https://github.com/Thinkmill/keystatic/tree/main/templates/astro)
```
Comment (sarah11918) [inline src/content/docs/en/guides/cms/craft-cms.mdx:15]: This link doesn't go to official documentation. Did you mean for this link to go to a template example? The documentation I could find seems to be here: 

```suggestion
- The official documentation for Craft's [`headlessMode` config setting](https://craftcms.com/docs/5.x/reference/config/general.html#headlessmode)
```

**Review by sarah11918 — COMMENTED** [review]
This looks great @olets , thank you for the contribution!

Just one link that looks a little off, but once that's resolved this should be good to publish!  :partying_face:

File: src/content/docs/en/guides/cms/craft-cms.mdx (hunk 1 above)
Comment (olets) [reply src/content/docs/en/guides/cms/craft-cms.mdx]: Good catch, no idea how that got into my clipboard. I don't remember even hearing of keystatic 🤔🤔

Updated

**Review by sarah11918 — APPROVED** [review]
Much better! :smile: 

Thanks for this great contribution, and welcome to Team Docs! :partying_face:

**Pull request merged**